content/en/docs/pipelines/installation/localcluster-deployment.md

```diff
@@ -1,48 +1,50 @@
 +++
-title = "Deploying Kubeflow Pipelines Standalone on a local cluster with kind, k3s, and k3s on WSL"
-description = "Instructions to deploy Kubeflow Pipelines Standalone to a local cluster for testing purposes"
+title = "Deploying Kubeflow Pipelines on a local cluster"
+description = "How to deploy Kubeflow Pipelines locally with kind, K3s, and K3ai for testing purposes"
 weight = 30
 +++
 
 This guide shows how to deploy Kubeflow Pipelines Standalone using:
 
 - kind
-- k3s
-- k3s on Windows Subsystem for Linux (WSL)
+- K3s
+- K3s on Windows Subsystem for Linux (WSL)
+- K3ai [*alpha*]
 
-Such deployment methods can be part of your local environment using the supplied 
-kustomize manifests for test purposes. This guide is an alternative to 
-[Deploying Kubeflow Pipelines (KFP)](/docs/started/getting-started/#installing-kubeflow).
+Such deployment methods can be part of your local environment using the supplied
+kustomize manifests for test purposes. This guide is an alternative to
+[Deploying Kubeflow Pipelines
+(KFP)](/docs/started/getting-started/#installing-kubeflow).
 
 ## Before you get started
 
 - You should be familiar with [Kubernetes](https://kubernetes.io/docs/home/),
-  [kubectl](https://kubernetes.io/docs/reference/kubectl/overview/), and 
+  [kubectl](https://kubernetes.io/docs/reference/kubectl/overview/), and
   [kustomize](https://kustomize.io/).
- 
-- You should have a Kubernetes cluster and install kubectl on it to work with 
-  Kubeflow Pipelines Standalone. 
- 
-- For native support of kustomize, you will need kubectl v1.14 or higher. You 
-  can download and install kubectl by following the 
-  [kubectl installation guide](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
 
-## kind 
+- You should have a Kubernetes cluster and install kubectl on it to work with
+  Kubeflow Pipelines Standalone.
```

Comment:
just realized you should probably remove this line

Reply:
/fixed and great catch!